ansible/files/postgres_prestart.sh.j2

```diff
@@ -19,31 +19,44 @@ get_shared_buffers() {
 update_orioledb_buffers() {
    local pg_conf="/etc/postgresql/postgresql.conf"
    local value="$1"
+   
+   # Update orioledb.main_buffers
    if grep -q "^orioledb.main_buffers = " "$pg_conf"; then
        sed -i "s/^orioledb.main_buffers = .*/orioledb.main_buffers = $value/" "$pg_conf"
    else
        echo "orioledb.main_buffers = $value" >> "$pg_conf"
    fi
-}
-
-main() {
-   local has_orioledb=$(check_orioledb_enabled)
-   if [ "$has_orioledb" -lt 1 ]; then
-       return 0
-   fi
-   local shared_buffers_value=$(get_shared_buffers)
-   if [ ! -z "$shared_buffers_value" ]; then
-       update_orioledb_buffers "$shared_buffers_value"
+   
+   # Update shared_buffers during alpha release to 32MB
+   # TODO will need to add logic here for different ec2 sizes
```

Comment:
Let's not duplicate that logic here. The right place to do it is in the internal binary that already generates these for our current setup.

Reply:
@darora ok I will look into how to add that appropriately in admin-api closing this